frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -24,16 +24,16 @@ page-section.c-section(:title='L("Invite links")')
         )
     tbody
       tr(
-        v-for='(item, index) in activeList'
-        :key='index'
+        v-for='(item, index) in invitesToShow'
+        :key='item.inviteSecret'
       )
         td.c-name
-          | {{ item.name }}
+          | {{ item.invitee }}
           tooltip.c-name-tooltip(
             v-if='item.isAnyoneLink'
             direction='top'
-            :isTextCenter='true'
-            :text='L("This invite link was only available during the onboarding period.")'
+            isTextCenter
```

Comment:
I'm noticing now that this is resulting in the following error in the console:

<img width="855" alt="Screen Shot 2019-11-22 at 7 59 06 PM" src="https://user-images.githubusercontent.com/138706/69472715-7e96f580-0d62-11ea-98e5-3ae8f95e6cf6.png">

This happened after I created a proposal to invite someone, and voted to generate the invite link, and then visited the Group Settings page.

And a totally separate issue is that the invite link that's generated from the proposal is being shown to everyone. I think it's supposed to only be shown to the user who created it? At least that's what I recall the designs saying, and the text says:

> Here's a list of all invite links you own

Reply:
fixed!